### PR TITLE
fix: decrease `maxQuotaMultiplier` to 2

### PR DESCRIPTION
**Pull request by lekhovitsky**

### CODE REVIEW

**Review by 0xmikko — APPROVED** [review]

File: contracts/test/integration/credit/Quotas.int.t.sol [hunk 1]
```diff
@@ -373,8 +373,8 @@ contract QuotasIntegrationTest is IntegrationTestHelper, ICreditManagerV3Events
         public
         creditTest
     {
-        _addQuotedToken(tokenTestSuite.addressOf(Tokens.LINK), 10_00, uint96(1_000_000 * WAD));
-        _addQuotedToken(tokenTestSuite.addressOf(Tokens.USDT), 500, uint96(1_000_000 * WAD));
+        _addQuotedToken(tokenTestSuite.addressOf(Tokens.LINK), 10_00, uint96(100_000 * WAD));
+        _addQuotedToken(tokenTestSuite.addressOf(Tokens.USDT), 500, uint96(100_000 * WAD));
```
Comment (3sGgpQ8H) [inline contracts/test/integration/credit/Quotas.int.t.sol:377]: These numbers should be named constants.

File: contracts/test/integration/credit/Quotas.int.t.sol [hunk 2]
```diff
@@ -408,8 +408,8 @@ contract QuotasIntegrationTest is IntegrationTestHelper, ICreditManagerV3Events
 
         (uint16 feeInterest,,,,) = creditManager.fees();
 
-        quotaLink = quotaLink > uint96(1_000_000 * WAD) ? uint96(1_000_000 * WAD) : quotaLink;
-        quotaUsdt = quotaUsdt > uint96(1_000_000 * WAD) ? uint96(1_000_000 * WAD) : quotaUsdt;
+        quotaLink = quotaLink > uint96(100_000 * WAD) ? uint96(100_000 * WAD) : quotaLink;
+        quotaUsdt = quotaUsdt > uint96(100_000 * WAD) ? uint96(100_000 * WAD) : quotaUsdt;
```
Comment (3sGgpQ8H) [inline contracts/test/integration/credit/Quotas.int.t.sol:412]: And these.

File: contracts/test/integration/credit/Quotas.int.t.sol [hunk 3]
```diff
@@ -441,7 +441,7 @@ contract QuotasIntegrationTest is IntegrationTestHelper, ICreditManagerV3Events
                 target: address(creditFacade),
                 callData: abi.encodeCall(
                     ICreditFacadeV3Multicall.updateQuota,
-                    (tokenTestSuite.addressOf(Tokens.LINK), int96(uint96(10000000 * WAD)), 0)
+                    (tokenTestSuite.addressOf(Tokens.LINK), int96(uint96(100_000 * WAD)), 0)
```
Comment (3sGgpQ8H) [inline contracts/test/integration/credit/Quotas.int.t.sol:444]: And also this.

**Review by 3sGgpQ8H — COMMENTED** [review]
It would be great to provide the reasoning for this change.

**Comment by lekhovitsky** [comment]
> It would be great to provide the reasoning for this change.

Multiplier of 8 was unjustifiably high. An account with multiple quotas of this size accrues quite a lot of debt in a form of quota interest and can quickly become unhealthy (under huge gas spikes, it can get into the bad debt zone before it's liquidated).

Combined with https://github.com/Gearbox-protocol/core-v3/pull/166, the value of 2 is sufficient to fully cover maximum debt and potential interest, and does not expose pools to the issue above that much.

**Pull request merged**